privacyidea/static/components/token/controllers/tokenDetailController.js

```diff
@@ -374,6 +374,8 @@ myApp.controller("tokenDetailController", ['$scope', 'TokenFactory',
                 params["machineid"] = machineObject.id;
                 params["resolver"] = machineObject.resolver;
             }
+            console.log("Attach machine");
+            console.log(params);
```

Comment:
is this necessary or just some debug info?

Reply:
Will remove it.

Reply:
Done in d8a89420